### DISCUSSION TITLE
Video plays back OK but does not fill the screen

### DISCUSSION THREAD

**Discussion by rgilton** [jellyfin-archive/jellyfin-roku-legacy#1813] [Q&A]
For some reason, Jellyfin on Roku only plays this video with _both_ vertical and horizontal letterboxing: i.e. it displays the video with a large black border around all sides.  Is there a setting that'll make it fill the screen?

The information on the video via the web interface:
```
Video
Title: 576p VP8 SDR
Codec: VP8
Profile: 0
Resolution: 720x576
Aspect ratio: 16:9
Interlaced: No
Framerate: 25
Bitrate: 2239 kbps
Bit depth: 8 bit
Video range: SDR
Video range type: SDR
Pixel format: yuv420p
Ref frames: 1
```

**Comment by cewert** [comment]
Are you still having this issue? If so, can you send me a copy of one of the files?

**Reply by rgilton** [reply]
Hi @cewert, yes I am still seeing the issue.  I have cut a few seconds out of the video to reduce the file size down -- so I now have a 3.4 MB file that still has the same problem.  How can I send you this file?

**Reply by cewert** [reply]
Pm me on matrix or you can email me. My username at gmail

**Reply by rgilton** [reply]
Sent (via email)

**Reply by cewert** [reply]
I was able to reproduce. I'll try looking into this
